cmake/compiler/host-clang/target.cmake

```diff
@@ -0,0 +1,20 @@
+# Configures CMake for using clang
+
+find_program(CMAKE_C_COMPILER   clang   )
+#find_program(CMAKE_CXX_COMPILER clang++ )    # Not used currently
+#find_program(CMAKE_OBJCOPY     objcopy )     # Not used currently
```

Comment:
Are you sure about that? I know a bunch of x86 post build steps use objcopy.
I'd rather we not commit commented-out code.

Reply:
@andrewboie 

I just hecked for native_posix, it’s used at CMake configure/generate step, but not at compile step.

Sure, let’s include it.